src/commands/config/analytics.ts

```diff
@@ -23,15 +25,23 @@ export default class Analytics extends Command {
 
       if (flags.disable) {
         analyticsConfigFileContent.analyticsEnabled = 'false';
-        this.log('Analytics disabled.');
+        this.log('\nAnalytics disabled.\n');
       } else if (flags.enable) {
         analyticsConfigFileContent.analyticsEnabled = 'true';
-        this.log('Analytics enabled.');
-      } else {
-        this.log('\nPlease append the "--disable" flag to the command in case you prefer to disable analytics, or use the "--enable" flag if you want to enable analytics back again.\n');
+        this.log('\nAnalytics enabled.\n');
+      } else if (!flags.status) {
+        this.log('\nPlease append the "--disable" flag to the command in case you prefer to disable analytics, or use the "--enable" flag if you want to enable analytics back again. In case you do not know the analytics current status, then you can append the "--status" flag to be aware of it.\n');
         return;
       }
       await writeFile(analyticsConfigFile, JSON.stringify(analyticsConfigFileContent), { encoding: 'utf8' });
+
+      if (flags.status) {
+        if (analyticsConfigFileContent.analyticsEnabled === 'true') {
+          this.log('\nAnalytics are enabled. Please append the "--disable" flag to the command in case you prefer to disable analytics.\n');
```

Comment:
```suggestion
          this.log('\nAnalytics are enabled.\n');
```
let's not encourage this, since we want the users to keep it enabled.